src/scss/components/main-nav/_main-nav.scss

```diff
@@ -0,0 +1,64 @@
+@charset "UTF-8";
+
+.su-multi-menu {
+  .su-multi-menu__nav-toggle--right {
+    &:hover::before,
+    &::before {
+      color: transparent;
+      background-color: transparent;
+    }
+
+    &[aria-expanded="true"] {
+      color: #e50808;
+
+      &::before {
+        color: #e50808;
+        background-color: #e50808;
+      }
+    }
+  }
+
+  @include grid-media-min('lg') {
+    > .su-multi-menu__menu > .su-multi-menu__item {
+      .su-multi-menu__link {
+        color: #2e2d29;
+
+        &:hover {
+          color: #8c1515;
+          text-decoration: none;
+
+          &::before {
+            color: #8c1515;
+            background-color: #8c1515;
+            transform: scaleX(1);
+            visibility: visible;
+          }
+        }
+      }
+
+      &.su-multi-menu__item--active-trail {
+        .su-multi-menu__link {
+          color: #e50808;
+
+          &::before {
+            color: #e50808;
+            background-color: #e50808;
```

Comment:
The :before element is being removed from the active li on hover; I think we need to adjust the javascript that is controlling this so it doesn't remove the :before element.